src/intents.js

```diff
@@ -46,6 +46,14 @@ function injectService (url, element, intent, data) {
         return event.source.postMessage(data, event.origin)
       }
 
+      if (handshaken && event.data.type === `intent-${intent._id}:size`) {
+        ['width', 'height', 'maxWidth', 'maxHeight', 'minWidth', 'minHeight'].forEach(prop => {
```

Comment:
I totally disagree because we will have code which is not used, but if I am alone let's go with it. (Just brace yourself for the day someone will ask "May I change `maxHeight` ? Where is it used ? What the difference with `height` ? 😄 )